tools/pihole_toml_to_markdown.py

```diff
@@ -0,0 +1,174 @@
+import re
+from pathlib import Path
+
+def parse_toml_with_comments(filepath):
+    with open(filepath, encoding='utf-8') as f:
+        lines = f.readlines()
+
+    documentation = ["""---
+description: This file is automatically generated when new code is pushed to the master branch of the FTL Repo. Any proposed changes should be made against https://github.com/pi-hole/ftl/tools/pihole_toml_to_markdown.py
+---
+
+# Pi-hole FTL Configuration Reference
+
+This page documents the available options in the `pihole-FTL.conf` file, which is typically managed via the TOML-formatted configuration file:
+
+```text\n/etc/pihole/pihole.toml\n```
+
+The file can be edited directly, however you can also use the command line option or the web interface.
```

Comment:
```suggestion
This page documents the available options of `pihole-FTL`. They are typically managed via the [TOML](https://toml.io/)-formatted configuration file `/etc/pihole/pihole.toml`. This file may be edited directly or you can use the command line (CLI) option, the web interface, the application programming interface (API) or environment variables.

Using the web interface, the API or the CLI is preferred as they can do error checking for you, trying to prevent any incompatible options which could prevent FTL from starting on a severely broken configuration.
```